gaj23/bigbirds
# Gaj/feature/home page

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. Include screenshots as needed.

This PR includes the completion of a static homepage for my app that also includes the beginning of testing as well. Router has been applied in certain areas, too.

<img width="300" alt="static homepage screen shot" src="https://user-images.githubusercontent.com/68332132/110145256-d8c49a00-7d9e-11eb-86d7-ee255f149f3e.png">

### Problems Encountered & Resulting Solutions  
> State any problems encountered & include screenshots as needed. 
> Describe your code changes in detail for reviewers.

At this time, no major problems were encountered. 

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?
 
See `App`, `Highlights`, and `Nav` in `src` for code details and psuedocode. 

See `cypress` folder for testing.

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Please clone `git@github.com:gaj23/bigbirds.git`, ensure npm is installed, and run `npm start` to view in browser. No buttons are functioning.

For tests, please ensure that you've downloaded cypress (`npm i -D cypress`) and that it exists in your `package.json` under scripts as `cypress: cypress open` before you run `npm run cypress` in your command line to view the tests. 

### Steps To Take
> Any questions and possible steps moving forward.

At this time, the highlights data has been hardcoded. It is a temporary solution until the api call occurs and data can be applied. Additionally, I will need to create mock data that contains image urls for display. I see this taking significant time. 


#### Checklist:
